keda-metrics-adapter/version.txt

```diff
@@ -0,0 +1 @@
+0.0.0
```

Comment:
@dimberman @sjmiller609 0.0.0 doesn't feel right to publish. Should we push a `1.0.0` or `0.0.1` or something?